### PR TITLE
Update baseUrl and sanitizeEupsTag

### PR DESCRIPTION
**Pull request by gcomoretto**
Please have a look.

I think that both changes are required to go to master.

### CODE REVIEW

File: pipelines/lib/util.groovy [hunk 1]
```diff
@@ -1411,6 +1411,12 @@ def String sanitizeEupsTag(String tag) {
   // eups doesn't like dots in tags, convert to underscores
   // by policy, we're not allowing dash either
   tag.tr('.-', '_')
+  // if the git tag is an official version, starts with a number
+  // but eups tag need still to have 'v' in front
+  char c = tag.charAt(0)
+  if ( c.isDigit() ) {
```
[the remark this answers is not in the document]
Comment (gcomoretto) [reply pipelines/lib/util.groovy]: OK. Updated as suggested.

File: pipelines/release/tarball.groovy [hunk 2]
```diff
@@ -790,7 +790,7 @@ def String smokeScript(
   MinicondaEnv menv,
   String ciDir
 ) {
-  def baseUrl = util.githubSlugToUrl('lsst/base')
+  def baseUrl = "https://raw.githubusercontent.com/${scipipe.versiondb.github_repo}"
```
[the remark this answers is not in the document]
Comment (gcomoretto) [reply pipelines/release/tarball.groovy]: Thank you for the hint.
I have updated the code to use the correct definitions from build_matrix.yaml.

File: pipelines/lib/util.groovy (hunk 1 above)
[the remark this answers is not in the document]
Comment (jhoblitt) [reply pipelines/lib/util.groovy]: My comment here was wrong -- I was thinking of the official release format and neglected that this is a generic method.  The nightly and weekly tag format (`d_` and `w_`) also need to be supported.  Your original implementation is better and is similar to what I implemented in `sqre-codekit`.

File: pipelines/release/tarball.groovy (hunk 2 above)
[the remark this answers is not in the document]
Comment (jhoblitt) [reply pipelines/release/tarball.groovy]: LGTM.

File: pipelines/lib/util.groovy (hunk 1 above)
[the remark this answers is not in the document]
Comment (gcomoretto) [reply pipelines/lib/util.groovy]: Reverted to the first implementation.

**Pull request merged**